INSTALL.md

```diff
@@ -54,189 +54,198 @@ to stop the application, run:
 make dev-stop
 ```
 
-## Manual setup on Mac OS X (10.13 - 12.3.1)
+## Manual setup on MacOS (12.5.1)
 
 ### Prerequisites
 
-#### Ruby version
+#### Package and runtime version management
 
-The project supports **2.6.x**.
+To add all required missing packages, make sure you have [Homebrew](https://brew.sh/) installed in your machine.
 
-Verify you have a proper version by running on your terminal:
-
-```bash
-ruby -v
+To manage the required runtimes, such as Ruby and Node.js, we recommend using [asdf](https://asdf-vm.com/guide/getting-started.html#global). Install it with brew:
+```
+brew install asdf
 ```
 
-> We recommend using [rbenv](https://github.com/rbenv/rbenv) or [rvm](https://rvm.io/) to install any ruby version.
+> Don't forget to add to your ~/.zshrc:
```

Comment:
Seems a bit specific, not everyone might be using ZSH. Maybe just point to https://asdf-vm.com/guide/getting-started.html#_3-install-asdf

Reply:
This instruction is included in brew's output so it's just a reminder. We could say something like this instead:
```suggestion
> Don't forget to check for any *Caveats* in `brew`'s output.
```

On the other hand, it is specific because ZSH is the default shell in macOS.
The link is already present in the second paragraph (click on 'asdf') so anybody with different setup can check specific steps for it.

Reply:
Oh, OK, I forgot that ZSH was default on macOS.